### DISCUSSION TITLE
Should we consider using `geom_forecast()` from the `forecast` package for vis?

### DISCUSSION THREAD

**Discussion by nickreich** [hubverse-org/.github#14] [Ideas]
Something that @elray1 has mentioned several times is the possibility of using a ggplot "geom" in hubvis. Recently @sbfnk mentioned that the `forecast` package has a `geom_forecast()` in it. Here is the documentation: https://pkg.robjhyndman.com/forecast/reference/geom_forecast.html

Maybe worth thinking about whether or not we can/should take advantage of this as a part of hubVis? I don't know how readily the tools/formats would translate, but maybe worth looking into.

**Comment by elray1** [comment]
I suggest moving this discussion over here: https://github.com/Infectious-Disease-Modeling-Hubs/hubVis/issues/22

**Comment by aaronger** [comment]
(Unsure if discussion moved yet)
My sense is that there is not going to be much support in `forecast` for the general problem we have of representing time series of predictive distributions.  The only facility for this that I can see at a first glance is this: https://github.com/robjhyndman/forecast/blob/1f537ebddf318050f7c5ed362bf92cf618041edc/R/ggplot.R#L2090

The `lineribbon` stat and geom from package `ggdist` seem like a closer match to our needs, but I have struggled even for those with the question of weather shoe-horning our `out_type_id`'s into the line_ribbon aesthetics is feasible and worthwhile.

**Reply by elray1** [reply]
Let's close this discussion and move all related comments to that issue.
